### ISSUE TITLE
Will this series continue?

### ISSUE THREAD

**Issue by ImOkapi** [cfenollosa/os-tutorial#243]
Please make this series continue, It helped me learn how to make an OS A LOT.
And i'm sure it will help a lot more people

**Comment by wantyapps** [comment]
I actually used this tutorial to help me understand how certain VGA
drivers and the kernel works. I do think it is based on
[OSDev](https://wiki.osdev.org) though.

**Comment by PauloDanielCarneiro** [comment]
Yes, that is stated in the readme... I imagine someone could fork it with the intention to continue the project. I think it would be hard for the original author to continue, as it have already 4 years since the last commit.

**Comment by wantyapps** [comment]
I actually made some changes to the kernel, but mainly private, so I think I can make a fork with some of the public ones. For example, I added verbosity and other stuff to the Makefile, added multiple makefiles for each module of the kernel (`libc`, `kernel` etc.). I also added comment-docs for almost every function in the kernel.c file (I actually kinda need help with that) and am working on a log feature like Linux's holy `dmesg`.
